packaging/pep517_backend/_backend.py

```diff
@@ -93,21 +115,37 @@ def build_sdist(  # noqa: WPS210, WPS430
     build_manpages_requested = BUILD_MANPAGES_CONFIG_SETTING in (
         config_settings or {}
     )
-    if build_manpages_requested:
-        Path('docs/man/man1/').mkdir(exist_ok=True, parents=True)
-        version_number = _get_package_distribution_version()
-        for rst_in in _generate_rst_in_templates():
-            _convert_rst_in_template_to_manpage(
-                rst_doc_template=rst_in.read_text(),
-                destination_path=rst_in.with_suffix('').with_suffix(''),
-                version_number=version_number,
+    original_src_dir = Path.cwd().resolve()
+    with _run_in_temporary_directory() as tmp_dir:
+        tmp_src_dir = Path(tmp_dir) / 'src'
+        copytree(original_src_dir, tmp_src_dir)
+        os.chdir(tmp_src_dir)
+
+        if build_manpages_requested:
+            Path('docs/man/man1/').mkdir(exist_ok=True, parents=True)
+            version_number = _get_package_distribution_version()
+            for rst_in in _generate_rst_in_templates():
+                _convert_rst_in_template_to_manpage(
+                    rst_doc_template=rst_in.read_text(),
+                    destination_path=rst_in.with_suffix('').with_suffix(''),
+                    version_number=version_number,
+                )
+                rst_in.unlink()
+
+        Path('pyproject.toml').write_text(
+            re.sub(
+                r'backend-path = \[[^\]]+\].*\nbuild-backend = "[^"]+".*\n',
```

Comment:
This regex can be simplified by removing the unnecessary escapes:

```suggestion
                r'backend-path = \[[^]]+].*\nbuild-backend = "[^"]+".*\n',
```

Reply:
@mattclay would `backend-path = ["stuff"]` still match if the closing `]` in the regex is unescaped but the opening `[` remains escaped? Regarding the internal `\]`, wouldn't it be more readable if it's clear that said `]` marked for exclusion is not the wrapping bracket?

Reply:
Yes, because `]` has no special meaning when it's outside `[]`. As for the internal `\]`, I find it more readable without the unnecessary escape. Perhaps that's in part because syntax highlighting in PyCharm shows the `]` has no special meaning in that scenario -- while including the leading `\` causes it to be highlighted as unnecessary.

Reply:
Interesting... I don't have any special highlighting for regexes in AstroNvim, maybe I need to see if there's something treesitter-based available. This is why it's harder for me to parse such a combination of chars.

I suppose people reading this in other editors or on GH, might get confused with it too.
So, I'll accept the suggestion but will make a follow-up improvement attempting to make it a bit more readable by multilining.